frontend.py

```diff
@@ -142,12 +142,19 @@ def runtime(state):
 				lineBreak(columns, getConfig(4))
 				printColor("What would you like to name your new note?",getConfig(2), "")
 				newNoteName = input(": ")
-				printColor("What content would you like to add to start? (If you wish to add content later, enter 'none'.)", getConfig(2),"")
-				newContent = input(": ")
-				if (newContent == 'none'):
-					apiCalls.createNote(newNoteName,"")
-				else:
-					apiCalls.createNote(newNoteName,newContent)
+				printColor("What content would you like to add to start? (If you wish to add content later, enter '*END'.)", getConfig(2),"")
+				printColor("*END: Stop typing to file.",getConfig(2),"")
+				printColor("return key: start a new line.",getConfig(2),"\n")
+				newContent1 = []
+				while True: 
+					notes = input("")
+					if notes == "*END":
```

Comment:
looks like we are going line by line? so to exit you have to type *END on its own line. shouldn't we say that it needs to be its own line in the user instructions above?

Reply:
I think it would be a good idea to specify that it needs to be on its own line. I can do that later today.